loader/generated/vk_loader_extensions.c

```diff
@@ -98,8 +98,6 @@ VKAPI_ATTR bool VKAPI_CALL loader_icd_init_entries(struct loader_icd_term *icd_t
     LOOKUP_GIPA(GetPhysicalDeviceSurfacePresentModesKHR, false);
 
     // ---- VK_KHR_swapchain extension commands
```

Comment:
This comment no longer makes sense.

Reply:
It does, vkGetPhysicalDevicePresentRectanglesKHR is 'from' VK_KHR_swapchain (as well as VK_KHR_device_groups)